search-service/src/main/kotlin/com/egm/stellio/search/entity/web/EntityOperationHandler.kt

```diff
@@ -109,7 +109,9 @@ class EntityOperationHandler(
         )
         @RequestParam queryParams: MultiValueMap<String, String>
     ): ResponseEntity<*> = either {
-        val options = queryParams.getFirst(QP.OPTIONS.key)
+        val options = queryParams.getFirst(QP.OPTIONS.key)?.split(",")
+        val disallowOverwrite = options?.any { it == OptionsValue.NO_OVERWRITE.value } == true
```

Comment:
I don't see a no_overwrite option for upsert in the specification.

Reply:
see the in-progress doc with misc fixes to contribute to the spec

Reply:
as discussed, aligned with the figures of operations from 1.6.1. Spec needs clarification, to be discussed with ISG-CIM.